docs/sql/index.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+title: SQL
+---
+
+# SQL Query Language
+
```

Comment:
Is this an alpha or beta feature? If so, then we need to note it at the top and let people know what to expect.